src/views/About.vue

```diff
@@ -8,6 +8,7 @@
         <BioCard name="Greg Vance" title="VP Finance" path="https://infosecucalgary.ca/uploads/greg-headshot.jpg" secret="Something Something Supreme"/>
         <BioCard name="Maryam Bouh" title="VP Operations" path="https://infosecucalgary.ca/uploads/maryam-headshot.jpg"  />
         <BioCard name="Marianna Ferreira" title="VP Communications" path="https://infosecucalgary.ca/uploads/marianna-headshot.jpg"  />
+        <BioCard name="Waleed Ghumman" title="VP Secretary" path="https://infosecucalgary.ca/uploads/marianna-headshot.jpg"  />
```

Comment:
currently the headshot for Waleed Ghumman is linked to Marianna's headshot

Reply:
Waleed is indeed not Marianna. I have fixed the error.